lib/internal/modules/esm/loader.js

```diff
@@ -193,6 +198,10 @@ class ModuleLoader {
     }
   }
 
+  setImportMap(importMap) {
+    this.#importMap = importMap;
+  }
+
```

Comment:
```suggestion
  set importMap(importMap) {
    this.#importMap = importMap;
  }

```

Reply:
But actually, is there any value hiding the class's importmap on a private field when the setter merely sets a new value blindly?

Reply:
Yeah, that was a later addition and I had the same thought as I was writing it. Originally I had passed it into the constructor which is when I setup the private field. Let me just remove and simplify all that.